### DISCUSSION TITLE
Convex mesh pivot position

### DISCUSSION THREAD

**Discussion by korokuchum** [NVIDIA-Omniverse/PhysX#348] [Q&A] [answered]
What would happen if geometry pivot of convex mesh is located outside of said geometry?
How well does PhysX (5.3.1 version to be specific) work with such geometry? And is it even legal to do this?

At the very least it works* in my case for the most part.
And I don't get any warnings/asserts/errors from PhysX sdk.
But my gut feeling is telling me this is a bad idea.

**Comment by vreutskyy** [comment, marked as answer]
It's totally legal and should work. Unless you make a millimeter scale mesh, hundreds of kilometers away from the origin. In which case you may get some floating-point precision issues.
